src/super_gradients/training/utils/module_utils.py

```diff
@@ -219,3 +220,30 @@ def __init__(self, mean_original, std_original, mean_required, std_required):
     def forward(self, x):
         x = (x + self.additive) * self.multiplier
         return x
+
+
+class UpsampleMode(Enum):
```

Comment:
please move this into a separate file under `src/super_gradients/common/data_types/enum`

Reply:
Done [8302bdf](https://github.com/Deci-AI/super-gradients/pull/349/commits/8302bdfb3d68f0fab4d04ba201d122f0184c0443)